index.html

```diff
@@ -2,276 +2,313 @@
 <html>
 
 <head>
-  <title>Checklist for the LSST DESC Meeting (Arizona 2020)</title>
-  <meta http-equiv="content-type" content="text/html; charset=utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" rel="stylesheet">
-  <style>
-    html {
-      font-family: Arial, Helvetica, sans-serif;
-      color: #333;
-    }
-
-    a:any-link {
-      color: rgb(6, 155, 255);
-    }
-
-    #page {
-      max-width: 1000px;
-      margin: 16px auto;
-      padding: 0 12px 200px 12px;
-    }
-
-    pre {
-      display: inline;
-      white-space: normal;
-      size: 95%;
-      margin: 0 6px;
-    }
-
-    @supports (zoom:2) {
-      input[type=checkbox] {
-        zoom: 2;
-      }
-    }
-
-    @supports not (zoom:2) {
-      input[type=checkbox] {
-        transform: scale(1.6);
-        margin: 5px;
-      }
-    }
-
-    .progress-container {
-      width: 100%;
-      height: 30px;
-      background-color: #EEE;
-      margin: 30px 0;
-      position: sticky;
-      top: -1px;
-    }
-
-    #progress {
-      width: 1%;
-      height: 100%;
-      float: left;
-      background-color: rgb(6, 155, 255);
-    }
-
-    .item {
-      margin: 20px 0;
-      line-height: 1.2;
-      font-size: 120%;
-      display: flex;
-    }
-
-    .box {
-      margin-right: 12px;
-      vertical-align: top;
-    }
-
-    .text {
-      vertical-align: top;
-      flex-grow: 1;
-    }
-
-    .done {
-      text-decoration: line-through;
-      color: #999;
-    }
+     <title>Checklist for the LSST DESC July Virtual Meeting</title>
+     <meta http-equiv="content-type" content="text/html; charset=utf-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" rel="stylesheet">
+     <style>
+     html {
+          font-family: sans-serif, Helvetica, Arial;
```

Comment:
`sans-serif` should be placed as the last item as a fallback option. If you prefer a different font, you can add the specific font name as the first item.

```suggestion
          font-family: Helvetica, Arial, sans-serif;
```